tests/spec/core/linter-rules/no-headingless-sections-spec.js

```diff
@@ -11,6 +11,11 @@ describe("Core Linter Rule - 'no-headingless-sections'", () => {
     });
   });
   const doc = document.implementation.createHTMLDocument("test doc");
+  beforeEach(() => {
+    while (doc.body.firstChild) {
```

Comment:
Nit: maybe leave a comment with a link back to this PR? The code on its own doesn’t really explain itself.